components/omega/doc/userGuide/IOStreams.md

```diff
@@ -114,6 +116,16 @@ a template can be:
    - Hours for a frequency every Freq hours (*not* Freq times per hour)
    - Minutes for a frequency every Freq minutes (*not* Freq times per minute)
    - Seconds for a frequency every Freq seconds (*not* Freq times per seconds)
+- **FileFreq:** An optional entry for including multiple time slices in a
+   single file (only supported for output presently)
+- **FreqUnits:** A field that, combined with the integer frequency,
+   determines the frequency of new files when multiple time slices are
+   included in a file. The file frequency must be less than or equal to
+   the data frequency. Acceptable values include all of the frequency values
+   listed above except the one-time values (eg OnStartup/Shutdown or AtTime).
+   The filename template should reflect the frequency and the time used for
+   the file will correspond to the template at the time the file is first
+   opened.
```

Comment:
What should these be set to if you just want all time entries to go to a single file?  Should they be omitted? Or set to a very large frequency?

Reply:
Does Omega write contents to these files on startup (before any time steps have run)?  Do we need an option for specifying whether to write on startup or not?

Reply:
@xylar yes, if you want all entries in a single file, you'll need a file frequency that's very large. And no, we currently don't have a write on startup option, though you could kludge it by defining a separate stream with freq OnStartup that writes to the same file.  Can you remind me what the use case is for writing on startup? Would you just need contents and metadata? Or would you actually want a full write?

Reply:
In MPAS-Ocean, it has been pretty standard to write out history files that include the initial state.  This is particularly important for diagnostic variables that aren't part of an initial condition or restart file.

But we can cross that bridge when we come to it.  I think there may be a few places in Polaris where we assume that the first output for test cases is the initial condition.  We'll have to read the initial condition instead, but that's easy.

Reply:
> @xylar yes, if you want all entries in a single file, you'll need a file frequency that's very large.

Is the fact that I didn't have `FileFreq` and `FileFreqUnits` perhaps why I got the errors I reported above?  I can try adding those and running again.

Reply:
Possibly, but there may be other bugs too. Worth a try.

And on the startup thing, it's not a huge deal to add since I already have the OnStartup flag internally. But it'll probably require the explicit flag and then not using it as freq units - requiring a bit of change in interpretation for one-time read/writes. Maybe it's clearer that way anyway.

Reply:
Don't feel like you have to include that change here.  It might be something we need down the road so it sounds good to keep it in mind.